src/CovidAppointmentTable.js

```diff
@@ -40,7 +40,7 @@ export function sortAndFilterData(
     onlyShowAvailable
 ) {
     const filteredData = onlyShowAvailable
-        ? data.filter((entry) => entry.hasAppointments)
+        ? data.filter((entry) => hasSignUpLink({ entry }))
```

Comment:
I think this is a bug - we need to remove the {} because then we're sending in `{entry: entry}` as the arg, while the function expects just plain old `entry` now

Reply:
Ok.  I guess I need to read up on curly brackets in arguments.

Reply:
Curly brackets for args are a short-hand way of saying this:
```function(entry) { //entry has properties "hasAppointments" and "timestamp" that we care about, for example
    const hasAppointments = entry.hasAppointments;
    const timestamp = entry.timestamp;
```
So the above is equivalent to
```
function({hasAppointments, timestamp}) { //now hasAppointments = entry.hasAppointments and timestamp =  entry.timestamp
```